test/install_test.go

```diff
@@ -19,11 +19,22 @@ import (
 ///   TEST SETUP   ///
 //////////////////////
 
-var TestHelper *testutil.TestHelper
+var (
+	TestHelper *testutil.TestHelper
+
+	// fetchLogsEvents is a switch to enable calling Testlogs and TestEvents
+	// when a test fails
+	fetchLogsEvents = false
```

Comment:
I think this should be descriptive rather than imperative.  i.e. it should be called `controlPlaneInstalled` or something and indicate that the control plane has been installed.